index.html

```diff
@@ -20,15 +20,15 @@
         <h1 class="name">Hey there. I’m<br>Mariane</h1>
         <h1 class="detail">I’m a software developer</h1>
         <p class="pOne">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <ul id="socials">
+        <li class="logos"><a href="#"><img src="./src/twitter.png" alt="twitter"></a></li>
+        <li class="logos"><a href="#"><img src="./src/linkedin.png" alt="linkedin"></a></li>
+        <li class="logos mediumlogo"><a href="#"><img src="./src/medium.png" alt="medium"></a></li>
+        <li class="logos"><a href="#"><img src="./src/github.png" alt="github"></a></li>
+        <li class="logos"><a href="#"><img src="./src/angellist.png" alt="angellist"></a></li>
+        </ul>
     </section>
-    <section id="socials">
-        <a href="#"><img src="./src/twitter.png" alt="twitter"></a>
-        <a href="#"><img src="./src/linkedin.png" alt="linkedin"></a>
-        <a class="mediumlogo" href="#"><img src="./src/medium.png" alt="medium"></a>
-        <a href="#"><img src="./src/github.png" alt="github"></a>
-        <a href="#"><img src="./src/angellist.png" alt="angellist"></a>
-    </section>
-    <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
+    <div class="temporary-spacer-div"></div>
 
 
 </body>
```

Comment:
- [ ] **REQUIRED:** The given live demo link on READMe.md file is leading to another branch's changes, I offer you to check the GitHub pages.